src/lib/select/select.ts

```diff
@@ -40,28 +40,31 @@ import 'rxjs/add/operator/startWith';
  * the trigger element.
  */
 
-/** The fixed height of every option element. */
-export const SELECT_OPTION_HEIGHT = 48;
+/** The fixed height of every option element (option, group header etc.). */
+export const SELECT_ITEM_HEIGHT = 48;
 
 /** The max height of the select's overlay panel */
 export const SELECT_PANEL_MAX_HEIGHT = 256;
 
 /** The max number of options visible at once in the select panel. */
-export const SELECT_MAX_OPTIONS_DISPLAYED = 5;
+export const SELECT_MAX_OPTIONS_DISPLAYED =
+    Math.floor(SELECT_PANEL_MAX_HEIGHT / SELECT_ITEM_HEIGHT);
```

Comment:
I switched to computing it, because it's easy to forget if we ever decide to change any of the other constants.